hello-world/rust/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
-name = "{{project-name}}"
```

Comment:
This file needs to keep the use of the templating name as people use it for `cosmo new`